packages/tuono/src/build/index.ts

```diff
@@ -17,6 +17,12 @@ const VITE_PORT = 3001
 function createBaseViteConfigFromTuonoConfig(
   tuonoConfig: TuonoConfig,
 ): InlineConfig {
+  /**
+   * @warning Keep in sync with {@link LazyLoadingPlugin} tests:
+   * packages/lazy-fn-vite-plugin/tests/transpileSource.test.ts
+   */
+  const pluginFilesInclude = /\.(jsx|js|mdx|md|tsx|ts)$/
```

Comment:
Since we have a monorepo, what do you think about creating a `constants|shared|i-dont-know-yet` folder to manage all the shared things?

Reply:
This is the first time I find myself "duplicating"**\*** code in the repo.
Right now I'm unaware of other redundant / duplicated content, so, from my point of view,
creating a new package right now adds only an overhead.

If you are aware of other scenarios like this,
I suggest to open a separate issue where we list all those cases,
then we decide if a new package is worth it and, eventually, how to name it.

**\*** I use `"` because the regexp resides here and in a test file so it's not duplicated in user-facing code)

Reply:
Agree. LGTM!